data/json/recipes/other/tool.json

```diff
@@ -128,5 +128,27 @@
       [ [ "pot_copper", 1 ] ],
       [ [ "hose", 2 ] ]
     ]
+  }, 
+  {
+    "result": "food_processor",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "skills_required": [ "fabrication", 4 ],
+    "time": 20000,
+    "reversible": true,
+    "autolearn": true,
+    "qualities": [
+      { "id": "SCREW", "level": 1, "amount": 1 },
+      { "id": "HAMMER", "level": 3, "amount": 1 }
+    ],
+    "tools": [ [ [ "soldering_iron", 20 ], [ "toolset", 20 ] ] ],
+    "components": [
+      [ [ "scrap", 4 ] ],
+      [ [ "cable", 2 ] ],
+      [ [ "pot", 1 ] ]
```

Comment:
But these contact switches must be deployed on some sort of a circuit board, not on a simple metal scrap, right? I'd say one or two pieces of `e_scrap` would suffice for both circuit board and contact switches.